### ISSUE TITLE
Move some INFO logging into a permanent log state

### ISSUE THREAD

**Issue by Fmstrat** [LemmyNet/lemmy#4725]
### Requirements

- [X] Is this a feature request? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a feature request? Do not put multiple feature requests in one issue.
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.
- [X] Do you agree to follow the rules in our [Code of Conduct](https://join-lemmy.org/docs/code_of_conduct.html)?

### Is your proposal related to a problem?

Certain log items are in `INFO`, specifically related to DB and upgrades, that should be output regardless of log mode. The example I have is this line:
```
INFO lemmy_db_schema::utils: Running Database migrations (This may take a long time)...
INFO lemmy_db_schema::utils: Database migrations complete.
```
Log items like these, that dictate the state of the system, should be output all the time, no matter what `RUST_LOG` is set to. For me, this led to a lot of debugging vs just being patient, since `lemmy-ui` didn't show that migrations were running (I only noticed from inspecting API calls to the INBOX that Lemmy backend was reporting startup). I can image others could run into the same issue.

There may be other areas this makes sense, but this one is a clear one (to me, anyway).

### Describe the solution you'd like.

Migration related log items, or any items that delay startup, should be logged no matter what `RUST_LOG` is set to.

### Describe alternatives you've considered.

Running `info` all the time. This seems overkill.

**Comment by dullbananas** [comment]
There's a lot of logs that shouldn't have info level, especially in request handlers.

**Comment by Nutomic** [comment]
@dullbananas I agree, individual requests should be logged as `trace`. However Im not even sure where the request logging is coming from, after commentening out [this](https://github.com/LemmyNet/lemmy/blob/main/src/lib.rs#L292) and [this](https://github.com/LemmyNet/lemmy/blob/main/src/lib.rs#L298) line it still logs all api requests as `info`.
